CONTRIBUTING.md

```diff
@@ -285,7 +285,7 @@ The reviewer may ask you to later squash the commits together to provide
 a clean commit history before merging in the feature.
 
 It's important to write a proper commit title and description. The commit title
-should fir around 50 characters; it is the first line of the commit text. The
+should fire around 50 characters; it is the first line of the commit text. The
```

Comment:
i have no idea what this was supposed to mean but it could probably be changed to `should be no more than 50 characters`